server/src/cm_cmd_exec.cpp

```diff
@@ -615,269 +626,41 @@ get_cubrid_mode_opt (T_CUBRID_MODE mode)
     return ("--" CSQL_CS_MODE_L);
 }
 
-static int
-parse_volume_line (T_SPACEDB_INFO * vol_info, char *str_buf)
-{
-    int volid, total_page, free_page;
-    char purpose[128], vol_name[PATH_MAX];
-    char *token = NULL;
-
-    volid = total_page = free_page = 0;
-    purpose[0] = vol_name[0] = '\0';
-
-    token = strtok (str_buf, " ");
-    if (token == NULL)
-    {
-        return FALSE;
-    }
-    volid = atoi (token);
-
-    token = strtok (NULL, " ");
-    if (token == NULL)
-    {
-        return FALSE;
+static int is_valid_database_description(char *str){
```

Comment:
These functions have true/false return values. We should use bool for return type.